spec/ruby_terraform/command_line/builder_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+require_relative '../../../lib/ruby_terraform/command_line/builder'
+require_relative '../../../lib/ruby_terraform/command_line/options'
+
+describe RubyTerraform::CommandLine::Builder do
```

Comment:
Again, I think these tests should be rewritten from the perspective of the command string they end up building such that `lino` becomes an implementation detail.